app/jobs/crons/migration_job.rb

```diff
@@ -0,0 +1,17 @@
+class Crons::MigrationJob < CronJob
+  queue_as :high_priority
+
+  include Sentry::Cron::MonitorCheckIns
+
+  # run at 12:30 AM every day
+  self.cron_expression = "30 0 * * *"
+
+  sentry_monitor_check_ins slug: "migration"
+
+  def perform
+    return unless Rails.application.config.npq_separation[:migration_enabled]
```

Comment:
does this mean we'll need to manually go in and check migrations vs github actions showing us if they passed/fail instead?

Reply:
also can we disable migration in staging/review otherwise we will start getting weird failures as it's enabled there?

Reply:
Also maybe better off using a feature flag instead so we can disable without having to 
make changes to config/re-deploy?

Reply:
I've updated to disable in non-migration/dev environments; I don't mind if we want to switch to a feature flag instead of config but suggest we pick that up separately 👍